cgrok/tree.h

```diff
@@ -0,0 +1,41 @@
+#include <stdbool.h>
```

Comment:
Header file for the shim that connects grok to libdict. This implements all the tokyocabinet functionality we need.